api/v1/kustomization_types.go

```diff
@@ -95,6 +99,14 @@ type KustomizationSpec struct {
 	// +required
 	Prune bool `json:"prune"`
 
+	// DeletionPolicy can be used to control garbage collection when this
+	// Kustomization is deleted. Valid values are ('MirrorPrune', 'Delete',
+	// 'Orphan'). 'MirrorPrune' mirrors the Prune field (false = 'Ophan',
+	// true = 'Delete'). Defaults to 'MirrorPrune'.
```

Comment:
If becoming a bit too technical and type-specific, writing it out in text can also work. Up to you!

```suggestion
	// 'Orphan'). 'MirrorPrune' mirrors the Prune field (orphan if false,
	// delete if true). Defaults to 'MirrorPrune'.
```